modules/openapi-generator/src/main/resources/python/api_test.mustache

```diff
@@ -28,6 +28,10 @@ class {{#operations}}Test{{classname}}(unittest.TestCase):
         {{{summary}}}  # noqa: E501
 {{/summary}}
         """
+{{#detectAmznApiGatewayExtensions}}
+        response = self.api.{{operationId}}()
+        print(response)
```

Comment:
What's this test doing?

Reply:
This test is invoking each method of the provided API. It is templated so that when the SDK is generated, it will just expand to "response = self.api.{name of method}()

Each method gets its own function from the api.mustache template.